docs/content/latest/yugabyte-platform/manage-deployments/remove-nodes.md

```diff
@@ -66,7 +66,7 @@ The action to release a node instance is available from the Removed internal sta
 Taking this action transfers the node to a BeingDecommissioned and then Decommissioned internal state, as follows:
 
 1. There is a wait for Master leader.
-2. This server is removed from blacklist on Master leader (blacklisting is described in [Remove node](#remove-node)).
+2. This server is removed from blacklist on Master leader (blacklisting is described in [Remove node](#remove-node/)).
```

Comment:
Link broken. Pretty sure you don't add trailing slash on in-page links

Reply:
fixed